helpers/error-message.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+module.exports = function (plugins) {
+  return function (message) {
+    return plugins.util.colors.red('\n' + '='.repeat(message.length) + ' \n')
```

Comment:
This leads to weird looking output on smaller console windows. I often stick console as a "sidebar" and have about ~200-300px width - in this case I'd have few lines of `=` before msg.